plugins/node/instrumentation-undici/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "@opentelemetry/instrumentation-undici",
+  "version": "0.1.0",
+  "description": "OpenTelemetry undici/fetch automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "prepublishOnly": "npm run compile",
+    "compile": "tsc -p .",
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "test-all-versions": "tav",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../../",
+    "watch": "tsc -w",
+    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/instrumentation-undici --include-dependencies",
+    "prewatch": "npm run precompile",
+    "version:update": "node ../../../scripts/version-update.js"
+  },
+  "keywords": [
+    "opentelemetry",
+    "fetch",
+    "undici",
+    "nodejs",
+    "tracing",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": "^1.7.0",
+    "@opentelemetry/sdk-metrics": "^1.8.0",
+    "@opentelemetry/sdk-trace-base": "^1.8.0",
+    "@opentelemetry/sdk-trace-node": "^1.8.0",
+    "@types/mocha": "7.0.2",
+    "@types/node": "18.6.5",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "rimraf": "5.0.5",
+    "semver": "^7.6.0",
+    "superagent": "8.0.9",
+    "test-all-versions": "6.0.0",
+    "ts-mocha": "10.0.0",
+    "typescript": "4.4.4",
+    "undici": "6.10.1"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.7.0"
+  },
+  "dependencies": {
+    "@opentelemetry/core": "^1.8.0",
+    "@opentelemetry/instrumentation": "^0.49.1"
```

I just released an update - sorry for the churn.
```suggestion
    "@opentelemetry/instrumentation": "^0.50.0"
```